.eslintrc.json

```diff
@@ -12,7 +12,7 @@
     "plugins": ["chai-friendly", "jsdoc", "prettier", "unicorn"],
     "parserOptions": {
         "sourceType": "module",
-        "ecmaVersion": 2020
+        "ecmaVersion": 2021
```

Comment:
What's the significance of adding ES module support?

Reply:
This, plus the addition of `@babel/eslint-parser` for `.mjs` files prevents lint errors for top-level await. The minimum Node version bump in package.json ensures it's supported at runtime. Theoretically it could be supported on Node 14.3.0+, but that would require an `--experimental-top-level-await` flag.